cudamapper/CMakeLists.txt

```diff
@@ -76,7 +76,11 @@ if (TARGET htslib_project)
 else()
      target_link_libraries(${MODULE_NAME} gwbase gwio cub)
 endif()
-target_compile_options(${MODULE_NAME} PRIVATE -Werror)
+
+target_compile_options(${MODULE_NAME} PRIVATE -Werror -Wall -Wextra)
+if (gw_optimize_for_native_cpu)
+    target_compile_options(cudapoa PRIVATE -march=native)
```

Comment:
Out of curiosity, why is the -march flag set for the cudapoa target here? Is it supposed to be `${MODULE_NAME}` or was this on purpose?

Reply:
Oh yes definitely needs to be `MODULE_NAME`. good catch!

Reply:
Fixed in latest version